ssh-key/src/certificate/options_map.rs

```diff
@@ -42,7 +42,16 @@ impl Decode for OptionsMap {
 
             while !reader.is_finished() {
                 let name = String::decode(reader)?;
-                let data = String::decode(reader)?;
+
+                // values can be anything, there's length encoding for whatever it is
+                let data_vec = Vec::decode(reader)?;
```

Comment:
This actually interprets the field as `bytes`, FWIW.

Since what you actually want to do here is read a length-prefixed field, use [`reader.read_prefixed`](https://docs.rs/ssh-encoding/latest/ssh_encoding/trait.Reader.html#method.read_prefixed)